### PR TITLE
Add tooltip for ListController

### PR DESCRIPTION
**Pull request by hoseinmirian**
Added tooltip for listController.vue and they are editable from json file listControllerData.json in data folder..

### CODE REVIEW

File: src/data/ListControllerData.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "gridList":  "Shows result in a grid way",
```
Comment (knirirr) [inline src/data/ListControllerData.json]:
```suggestion
  "gridList":  "Show results as a grid.",
```

File: src/data/ListControllerData.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  "stackList":  "Shows result in a stack way",
```
Comment (knirirr) [inline src/data/ListControllerData.json]:
```suggestion
  "stackList":  "Show results as a stack.",
```

File: src/data/ListControllerData.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  "sort":  "Sorts result based on different fields"
```
Comment (knirirr) [inline src/data/ListControllerData.json]:
```suggestion
  "sort":  "Sort results by different fields."
```

**Comment by knirirr** [comment]
This looks fine but I've made some suggestions for the tooltip text.

**Review by knirirr — APPROVED** [review]

**Comment by hoseinmirian** [comment]
@knirirr this branch was approved but now is back to review. Please let me know I need to change anything. I think review request was asking due to the new suggested commit

**Review by knirirr — APPROVED** [review]

**Pull request merged**